src/functionalTest/resources/application-functional.yaml

```diff
@@ -13,7 +13,7 @@ idam.auth.clientId: rd-professional-api
 
 idam.auth.tokenAuth: ${IDAM_TOKEN_ATUH:cmQtcHJvZmVzc2lvbmFsLWFwaTpjYzVmMmE2LTk2OTAtMTFlOS1iYzQyLTUyNmFmNzc2NGY2NA==}
 idam.auth.redirectUrl: ${OAUTH2_REDIRECT_URI:https://rd-professional-api-aat.service.core-compute-aat.internal/oauth2redirect}
```

Comment:
I think oauth2.secret and IDAM_TOKEN_ATUH can also be removed as it is unused, please check.

Reply:
i am not doing code cleanup now...later we can do